### ISSUE TITLE
Firmware?

### ISSUE THREAD

**Issue by dimecho** [roboterclubaachen/usb2can#5]
A bit confused about which firmware to use? Tried "slcan CANable firmware" but it builds for STM32F042x6. DFU flashes it but CAN not working. Is there a .hex or .bin for this hardware?

**Comment by daniel-k** [comment]
Have a look at https://github.com/nexulm/candleLight_fw/blob/fa7a75c04ef5fbaab253ac6e34735fadccb97d17/Makefile#L63-L67, @nexulm seems to be using the hardware and made the required changes to get it working with candleLight. Would be great to get some feedback if you try it out :)

**Comment by dimecho** [comment]
Thanks,

I changed 2 things for STM32F072xB

`ASM_SRC  = system/src/cmsis/startup_stm32f072xb.S`

and

`usb2can:
          $(MAKE) CHIP=STM32F072xB BOARD=usb2can bin`

Compiled OK. 

Bad news as soon as I pressed RST button magic smoke and chip burned - both red and blue LEDs stayed on. Unless I soldered something incorrectly the hardware is not 100%, did anyone actually build this? Why would STM32 burn by pressing RST?
